tests/build.gradle

```diff
@@ -4,21 +4,21 @@ apply plugin: 'realm-android'
 
 android {
     compileSdkVersion sdkVersion
-    buildToolsVersion buildTools
+    buildToolsVersion '28.0.2'
```

Comment:
Please update the variable value instead of replacing it with a hardcoded value.

Reply:
Hi @cmelchior i replaced buildToolsVersion  hardCoded in my last commit